makefile

```diff
@@ -214,5 +214,4 @@ testAllCommands:
 > $(MAKE) HelloWorld-node
 > $(MAKE) HelloWorld-build
 > $(MAKE) HelloWorld-buildWeb
-> $(MAKE) HelloWorld-testCI
```

Comment:
We can re-enable these per-recipe `-testCI` rules, but these might not add a lot of value beyond `testAllCI`.

Reply:
Yeah good point.